screen-orientation/onchange-event-subframe.html

```diff
@@ -25,5 +28,6 @@
     assert_equals(message.data, orientation, "subframe receives orientation change event");
```

Comment:
Curious; if this asserts here, what state is the browser left in (we don't reach `screen.orientation.unlock()` or `document.exitFullscreen()`, right?)? Could this affect subsequent tests (@foolip ?)

Reply:
`t.add_cleanup` is used in many fullscreen tests and would be a good idea here.

Reply:
Should this then try/catch and call t.add_cleanup?

Reply:
No try/catch. Add `t.add_cleanup` as early as possible (right after requesting fullscreen). It will run the cleanup when the test is considered done, regardless of test result.

Reply:
I didn't trust bless no to throw :) ... so I put the cleanup at the top of each test. Hope that's ok.